taxi/views.py

```diff
@@ -87,3 +88,26 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
```

Comment:
Consider using `get_user_model()` instead of directly referencing `Driver`. This change is necessary to ensure flexibility and adherence to best practices as suggested in the checklist.